core/promise/methods/noop/promise_issuer_test.go

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (C) 2018 The "MysteriumNetwork/node" Authors.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package noop
+
+import (
+	"errors"
+	"testing"
+
+	"github.com/mysteriumnetwork/node/core/connection"
+	"github.com/mysteriumnetwork/node/core/promise"
+	"github.com/mysteriumnetwork/node/identity"
+	"github.com/mysteriumnetwork/node/logconfig"
+	"github.com/mysteriumnetwork/node/money"
+	"github.com/mysteriumnetwork/node/service_discovery/dto"
+	"github.com/stretchr/testify/assert"
+)
+
+var (
+	providerID = identity.FromAddress("provider-id")
+	proposal   = dto.ServiceProposal{
+		ProviderID: providerID.Address,
+	}
+)
+
+var _ connection.PromiseIssuer = &PromiseIssuer{}
+
+func TestPromiseIssuer_Start_SubscriptionFails(t *testing.T) {
+	dialog := &fakeDialog{
+		returnError: errors.New("reject subscriptions"),
+	}
+	issuer := &PromiseIssuer{dialog: dialog}
+
+	logs := make([]string, 0)
+	logger := logconfig.ReplaceLogger(logconfig.NewLoggerCapture(&logs))
+	defer logconfig.ReplaceLogger(logger)
+
+	err := issuer.Start(proposal)
+	assert.EqualError(t, err, "reject subscriptions")
```

Comment:
define the errors.New("...") as a variable and use it here instead of the hardcoded string